checkov/sca_package/runner.py

```diff
@@ -134,14 +138,17 @@ def parse_vulns_to_records(self, report, result, rootless_file_path, runner_filt
             report.add_record(record)
 
     def find_scannable_files(
-        self, root_path: Optional[Path], files: Optional[List[str]], excluded_paths: Set[str], exclude_package_json: bool = True
+            self, root_path: Optional[Path], files: Optional[List[str]], excluded_paths: Set[str],
+            exclude_package_json: bool = True
     ) -> Set[Tuple[Path, Path]]:
         input_output_paths: Set[Tuple[Path, Path]] = set()
         if root_path:
             input_paths = {
                 file_path
                 for file_path in root_path.glob("**/*")
-                if file_path.name in SUPPORTED_PACKAGE_FILES and not any(p in file_path.parts for p in excluded_paths)
+                if (not any(p in file_path.parts for p in excluded_paths) and
+                    (file_path.name in SUPPORTED_PACKAGE_FILES or
+                     any(file_path.name.endswith(ext) for ext in SUPPORTED_PACKAGE_EXTENSIONS)))
```

Comment:
```suggestion
                     file_path.name.endswith(SUPPORTED_PACKAGE_EXTENSIONS)))
```
when you change `SUPPORTED_PACKAGE_EXTENSIONS` to a `tuple`, then you can directly pass it into `endswith`